FWCore/ParameterSet/python/TreeCrawler.py

```diff
@@ -126,6 +126,138 @@ def load_module(self, fqname, fp, pathname, (suffix, mode, type)):
+                                    # If control got this far without raising an exception, then it must be a valid python module
+                                    yield "import", (None, loadMethodArgument)
+                                except ImportError :
+                                    # Didn't work in the local area, try in the global area.
+                                    parentFilename=[self._globalarea+"/python"]
+                                    for subModule in loadMethodArgument.split(".") :
+                                        moduleInfo=imp.find_module( subModule, parentFilename )
+                                        parentFilename=[moduleInfo[1]]
+                                    # If control got this far without raising an exception, then it must be a valid python module
+                                    yield "import", (None, loadMethodArgument)
+                            except Exception as error:
+                                # Either there was an import error (not a python module) or there was a string
+                                # manipulaton error (argument not a string). Assume this wasn't a call on a
+                                # cms.Process object and move on silently.
+                                pass
+                        
+                        elif code[3]==LOAD_NAME :
+                            # The argument is a variable. I can get the name of the variable quite easily but
+                            # not the value, unless I execute all of the opcodes. Not sure what to do here,
+                            # guess I'll just print a warning so that the user knows?
+                            print "Unable to determine the value of variable '"+names[indexInTable]+"' to see if it is a proces.load(...) statement in file "+co.co_filename
+                        
+                        code=code[9:]
+                        continue
+
+            if c in STORE_OPS:
+                oparg, = unpack('<H', code[1:3])
+                yield "store", (names[oparg],)
+                code = code[3:]
+                continue
+            if code[:9:3] == LOAD_LOAD_AND_IMPORT:
+                oparg_1, oparg_2, oparg_3 = unpack('<xHxHxH', code[:9])
+                level = consts[oparg_1]
+                if level == -1: # normal import
+                    yield "import", (consts[oparg_2], names[oparg_3])
+                elif level == 0: # absolute import
+                    yield "absolute_import", (consts[oparg_2], names[oparg_3])
+                else: # relative import
+                    yield "relative_import", (level, consts[oparg_2], names[oparg_3])
+                code = code[9:]
+                continue
+            if c >= HAVE_ARGUMENT:
+                code = code[3:]
+            else:
+                code = code[1:]
+
+def removeRecursiveLoops( node, verbose=False, currentStack=None ) :
```

Comment:
By the way, one can also get it to report recursive loops (like the ones mentioned my comments) by changing `verbose` to True.  I defaulted it to False because it can throw up a lot of erroneous loops, and I thought general users would worry without reason.